addon/data/index.js

```diff
@@ -6,12 +6,12 @@ import { decoratorWithKeyReflection } from '../utils/decorator-macros';
  * Decorator that turns the property into an Ember Data attribute
  *
  * ```javascript
- * import DS from 'ember-data';
+ * import Model from 'ember-data/model';
  * import { attr } from 'ember-decorators/data';
  *
- * export default DS.Model.extend({
- *   @attr firstName
- * });
+ * export default class extends Model {
```

Comment:
Agreed, we should keep these examples as normal Ember Objects until we iron out their usage